disperser/dataapi/v2/server_v2.go

```diff
@@ -31,6 +32,13 @@ import (
 var errNotFound = errors.New("not found")
 
 const (
+	// The blob feed API queries blobs in time range [start, end), this constant
+	// sets the max length of this interval.
+	// It's set to 4h (in seconds), so if the Disperser is operating at the max
+	// request rate (100 blobs/s), there will be 1,440,000 blobs (14400*100) at max
```

Comment:
Is 1 million blobs too much?

Reply:
time range isn't a good way to cap the size, dropped it